### PR TITLE
Show OpenStack personalized credential warning as hint

### LINKED ISSUE #595: Make openstack warning for personalized credentials more visible
In secret dialog for OpenStack the warning `Do not use personalized login credentials. Instead, use credentials of a technical user.` is not visible enough, see the screenshot. 

Please put it on the top of the dialog and use some more-special formatting so that it will be the first thing that a user will read.

image

### PR DESCRIPTION
**Pull request by grolu**
**What this PR does / why we need it**:
Show OpenStack personalized credential warning as hint, currently not persistent
Improved Colorize Component to not overwrite error color classes
Renamed `SelectColorizer` to `Colorizer` Component as it is a universal component which can be used to colorize different kind of input elements

**Which issue(s) this PR fixes**:
Fixes #595

**Special notes for your reviewer**:
Screen Shot 2020-02-27 at 21 43 57

**Release note**:

```improvement user
NONE
```

### CODE REVIEW

**Comment by grolu** [comment]
cc @vpnachev

File: frontend/src/components/HintColorizer.vue [hunk 1]
```diff
@@ -21,13 +21,21 @@ limitations under the License.
...
-  name: 'select-hint-colorizer',
+  name: 'hint-colorizer',
   props: {
     hintColor: {
       type: String
     }
   },
+  computed: {
+    isSelectErrorColor () {
+      const color = get(this, '$children[0].$children[0].color')
```
Comment (petersutter) [inline frontend/src/components/HintColorizer.vue]: do we have a test for this?

Comment (petersutter) [reply frontend/src/components/HintColorizer.vue]: just saw that a test exists 👍

File: frontend/tests/unit/HintColorizer.spec.js [hunk 2]
```diff
@@ -39,4 +39,19 @@ describe('SelectHintColorizer.vue', function () {
...
     expect(colorizerComponent.$el.className).to.not.contain('hintColor-cyan')
   })
+
+  it('should not overwrite error color class', async function () {
+    const data = () => {
+      return {
+        errorMessage: undefined
+      }
+    }
+    const template = '<hint-colorizer hintColor="orange" ref="hintColorizer"><v-text-field :error-messages="errorMessage"></v-text-field></hint-colorizer>'
```
Comment (petersutter) [inline frontend/tests/unit/HintColorizer.spec.js]: add test for v-select

**Comment by vpnachev** [comment]
Does the password have similar warning ?

**Comment by grolu** [comment]
> Does the password have similar warning ?
@vpnachev No, which case do you want to catch with that? If the user uses a technical account, then the password is (hopefully) different from the one he uses for his personal account anyway

**Comment by vpnachev** [comment]
You are right, but even if the user types his personal ID, maybe he will re-consider doing it once he sees one more warning for the password field.

Also, could you increase the font and make it red ?

**Comment by grolu** [comment]
I added a hint for the password field, too. We don't want to show the hint with red color, as this usually indicates an error for the input field. The user would probably not understand this. We already show the hint in warning color which should be sufficient. Font size and weight is the same form all hints on the UI and it would be a b ad idea from ux perspective to change it only for this use case.
If we continue to have issues with this after we released the changes, we can have another discussion.

**Pull request merged**